htdocs/admin/mails_templates.php

```diff
@@ -540,7 +540,7 @@
 
 // Show fields for topic, join files and body
 $fieldsforcontent = array('topic', 'joinfiles', 'content');
-if (!empty($conf->global->MAIN_EMAIL_TEMPLATES_FOR_OBJECT_LINES)) { $fieldsforcontent = array('content', 'content_lines'); }
+if (!empty($conf->global->MAIN_EMAIL_TEMPLATES_FOR_OBJECT_LINES)) { $fieldsforcontent = array('topic', 'content', 'content_lines'); }
```

Comment:
Why removing 'joinfiles' ?

Reply:
I have not checked if joifiles should be.